### ISSUE TITLE
【Epic】フェーズ2: AIエンジン開発 - データ準備とモデル構築

### ISSUE THREAD

**Issue by HappyMana** [HappyMana/proofreading_app#8]
## 概要
文章校正システムのAI機能を実現するため、機械学習モデルの訓練用データ準備とモデル構築を行います。

## 目標
- 高品質な日本語校正データセットの構築
- 日本語BERTベースの校正モデル開発
- 高精度な誤り検出・修正機能の実現

## 含まれるタスク
- [ ] 日本語校正データ収集・整理
- [ ] アノテーション作業・品質管理
- [ ] 訓練/検証/テストデータ分割
- [ ] 日本語BERTファインチューニング
- [ ] 誤り検出モデル構築
- [ ] 誤り修正モデル構築
- [ ] モデル評価・改善

## 期間
6-8週間

## 完了条件
- 10万件以上の高品質校正データセット完成
- 誤字脱字検出率95%以上のモデル
- 文法エラー検出率90%以上のモデル
- レスポンス時間1秒以内（1000文字あたり）

## 技術スタック
- **ML Framework**: PyTorch, HuggingFace Transformers
- **Model**: 日本語BERT (cl-tohoku/bert-base-japanese)
- **Data Processing**: pandas, numpy
- **Evaluation**: scikit-learn

## 関連issue
このepicに関連する詳細タスクissueは順次作成予定

🏷️ Labels: epic, phase-2, enhancement, machine-learning

**Comment by HappyMana** [comment]
新アーキテクチャではAI API活用に変更のため、独自モデル訓練は不要となりました。
